src/components/Content.tsx

```diff
@@ -114,6 +114,7 @@ const Content: ContentComponent = props => {
         ? <Search />
         : <React.Fragment>
           {rootThoughtsLength === 0 ? <NewThoughtInstructions childrenLength={rootThoughtsLength} isTutorial={isTutorialLocal} /> : <Subthoughts
+            thoughtsResolved={[]}
```

Comment:
Note that `[]` is not a valid `Path`; a `Path` must have length of at least 1. (Unfortunately the type checker is not able to catch this.)